doc/md/guides/SQLite/partial-indexes-sqlite.md

```diff
@@ -0,0 +1,263 @@
+---
+id: partial-indexes-sqlite
+title: Partial Indexes in SQLite
+slug: /guides/SQLite/partial-indexes-sqlite
+---
+
+### Overview of Partial Indexes
+
+#### What are Partial Indexes?
+
+With SQLite, users may create _partial indexes_, which are types of indexes that exist on a subset of a table, rather than the entire table itself. If used correctly, partial indexes improve performance and reduce costs, all while minimizing the amount of storage space they take up on the disk.
+
+#### Why do we need them?
+
+Let's demonstrate a case where partial indexes may be useful by contrasting them with a non-partial index. ​​If you have many records in an indexed table, the number of records the index needs to track also grows. If the index grows in size, the disk space needed to store the index itself increases as well.
+In many tables, different records are not accessed with uniform frequency. A subset of a table's records might not be searched very frequently or not searched at all. Records take up precious space in your index whether they are queried or not, and are updated when a new entry is added to the field.
+
+Partial indexes come into the picture to filter unsearched values and give you, as an engineer, a tool to index only what's important.
+
+#### Advantages of using Partial Indexes
+In cases where we know ahead of time the access pattern to a table and can reduce the size of an index by making it partial:
+1. Response time for SELECT operations is improved because the database searches through a smaller index.
+2. On average, response time for UPDATE operations is also improved as the index is not going to get updated in all cases.
+3. Index is smaller in size and can fit into RAM more easily.
+4. Less space is required to store the index on disk.
```

Comment:
We can use a simplified version of it (just to not make it a copy of PostgreSQL), and use some terms from the first section here: https://www.sqlite.org/partialindex.html